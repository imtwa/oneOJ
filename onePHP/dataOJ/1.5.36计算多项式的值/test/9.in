19.5618 17