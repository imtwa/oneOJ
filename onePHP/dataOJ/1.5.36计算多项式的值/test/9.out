9478860707306928078848.00